src/main/java/uk/gov/hmcts/reform/sendletter/logging/AppInsights.java

```diff
@@ -34,6 +41,24 @@ public void trackMessageAcknowledgement(java.time.Duration duration, boolean suc
         }
     }
 
+    public void trackNotPrintedLetter(NotPrintedLetter notPrintedLetter) {
+        java.time.Duration sinceCreated = java.time.Duration.between(notPrintedLetter.createdAt, ZonedDateTime.now());
+
+        telemetry.trackEvent(
+            LETTER_NOT_PRINTED,
+            ImmutableMap.of(
+                "letterId", notPrintedLetter.id.toString(),
+                "messageId", notPrintedLetter.messageId,
+                "service", notPrintedLetter.service,
+                "type", notPrintedLetter.type,
+                "sentToPrintAt", notPrintedLetter.sentToPrintAt.toString()
+            ),
+            Collections.singletonMap(
+                "daysCreated", ((double) sinceCreated.getSeconds()) / SECONDS_PER_DAY
```

Comment:
why not `.toDays()`?

Reply:
want to get use of enforced `double` in metrics